modules/security-context-constraints-psa-synchronization.adoc

```diff
@@ -7,13 +7,11 @@
 [id="security-context-constraints-psa-synchronization_{context}"]
 = Security context constraint synchronization with pod security standards
 
-{product-title} includes link:https://kubernetes.io/docs/concepts/security/pod-security-admission[Kubernetes pod security admission]. Globally, the `privileged` profile is enforced, and the `restricted` profile is used for warnings and audits.
-
 In addition to the global pod security admission control configuration, a controller exists that applies pod security admission control `warn` and `audit` labels to namespaces according to the SCC permissions of the service accounts that are in a given namespace.
 
 [IMPORTANT]
 ====
-Namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. You can enable pod security admission synchronization on other namespaces as necessary. If an Operator is installed in a user-created `openshift-*` namespace, synchronization is turned on by default after a cluster service version (CSV) is created in the namespace. 
+Namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. You can enable pod security admission synchronization on other namespaces as necessary. If an Operator is installed in a user-created `openshift-*` namespace, synchronization is turned on by default after a cluster service version (CSV) is created in the namespace.
 ====
 
 The controller examines `ServiceAccount` object permissions to use security context constraints in each namespace. Security context constraints (SCCs) are mapped to pod security profiles based on their field values; the controller uses these translated profiles. Pod security admission `warn` and `audit` labels are set to the most privileged pod security profile found in the namespace to prevent warnings and audit logging as pods are created.
```

Comment:
```suggestion
The controller examines `ServiceAccount` object permissions to use security context constraints in each namespace. Security context constraints (SCCs) are mapped to pod security profiles based on their field values; the controller uses these translated profiles. Pod security admission `warn` and `audit` labels are set to the most privileged pod security profile in the namespace to prevent warnings and to audit logging as pods are created.
```
- I think you can remove "found" and retain the intended meaning.
- I added the "to" before "audit" because my brain kept expecting to read "to prevent warnings and audits", even though that wasn't your wording. Might just be me, but I thought I'd mention it.

Reply:
I think we can remove "found" here, but the "to" in the second item doesn't retain the right meaning. But will play with it some, thanks!

Reply:
@lahinson I reworded this to "Pod security admission `warn` and `audit` labels are set to the most privileged pod security profile in the namespace to prevent displaying warnings and logging audit events when pods are created. "

Let me know if that sounds clearer

Reply:
Nice! That's clearer 😄